### PR TITLE
Accessability

### PR DESCRIPTION
**Pull request by c00p75**
- Added alternative image text
- Resized fonts
- Used appropriate headers

### CODE REVIEW

**Comment by solorics** [comment]
### **Portfolio Accessibillity** 💯 👍 

- Add form label to Portfolio
- Fix linter Errors ✅

File: index.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 					<a href="#" class="hover_effect">George M'sapenda</a>
 				</div>
 				<button class="btn">
-					<img src="images/Menu_icon.png" alt="menu bar icon">
+					<img src="/images/Icon - Menu.png" alt="menu">
 				</button> 
 				<ul id="desktop_nav">
 					<li><a href="#" class="hover_effect">Portifolio</a></li>
```
Comment (christianbiring1) [inline index.html:22]: [OPTIONAL]
- [ ] You may consider adding some spacing between your nav links 
Your design :point_down: 
image
Figma design :point_down: 
image

File: index.html [hunk 2]
```diff
@@ -234,56 +234,54 @@ <h2 class="about_me_header">About<br>Myself</h2>
...
 										<span class="language_name">HTML</span>
 									</div>
 								</li>				
 								<li class="language">
 									<div class="lan">
-										<img src="images/css.png" alt="css icon">
+										<img src="images/css.png" alt="css">
 										<span class="language_name">CSS</span>
 									</div>
 								</li>
```
Comment (christianbiring1) [inline index.html:287]: [OPTIONAL]
- [ ] Kindly consider fixing the size of all the buttons on your project cards

Your design :point_down: 
image

Figma design :point_down: 
image

File: index.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
 					<a href="#" class="hover_effect">George M'sapenda</a>
 				</div>
 				<button class="btn">
-					<img src="images/Menu_icon.png" alt="menu bar icon">
+					<img src="/images/Icon - Menu.png" alt="menu">
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure you use the correct path for your hamburger menu image.
Your design :point_down: 
image

Figma template :point_down: 
image

File: style.css [hunk 4]
```diff
@@ -878,10 +869,6 @@ li {
     width: 3vw;
   }
 
-  .language_name {
-    font-size: 1vw;
-  }
-
   .contact_section_container {
```
Comment (christianbiring1) [inline style.css:874]: [OPTIONAL]
- [ ] You may also consider readjusting the spacing of your form. 

Your design :point_down: 
image

Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @c00p75 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
almost


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good UI :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by solorics** [comment]
### **Reviewer's review** 💯 

- Worked on the Reviewer's review 
-image

Screenshot 2022-11-09 at 23 11 52 🔥 
- No Linters errors 👍

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @c00p75 

Your project is complete! Great job for making the changes requested 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- well accesible
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**